src/base/static/models/place.ts

```diff
@@ -0,0 +1,26 @@
+export type PlaceTag = {
```

Comment:
FYI I noticed that we were starting to import types between ducks, which can lead to circular imports. To avoid that, I started a "models" directory. I'm not sure how we want to structure this directory just yet, but I think it will come in handy as a way to consolidate our serializer and util methods.